custom_image_utils/args_parser.py

```diff
@@ -219,6 +219,13 @@ def parse_args(args):
       instance. This value may need to be increased if your init script
       generates a lot of output on stdout. If not specified, the default value
       of 300 seconds will be used.""")
+  parser.add_argument(
+      "--licenses",
+      type=str,
+      required=False,
+      default="",
+      help=
+      """(Optional) Comma-separated list of URIs to license resources.""")
```

Comment:
this is a little-bit fragile - if someone writes `--licenses a, b, c`, this will break.

`argparse` has a nice way to handle optional variable arg lists: specify `nargs='*'` and the user can provide a space-separated list  (for example: `--licenses a b c`).  Then, you'll get a list of those strings in `licenses`, which you can `','.join(licenses)` when you pass it into the template.

That'll handle little changes in command line formatting better.

Reply:
We talked about this out of band - since this code is passing through the format gcloud requires, I think it's okay to break when gcloud breaks.